zeppelin-web/src/app/notebook/paragraph/result/result.controller.js

```diff
@@ -176,7 +176,8 @@ function ResultCtrl ($scope, $rootScope, $route, $window, $routeParams, $locatio
       $scope.builtInTableDataVisualizationList.push({
         id: vis.id,
         name: vis.name,
-        icon: $sce.trustAsHtml(vis.icon)
+        icon: $sce.trustAsHtml(vis.icon),
+        supports: [DefaultDisplayType.TABLE, DefaultDisplayType.NETWORK]
```

Comment:
Some notes for other reviewers. 

`supports` field was originated from https://github.com/apache/zeppelin/pull/2125/files.